server/models/Conversation.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+
+const conversationSchema = new mongoose.Schema({
+  messages: [{ type: mongoose.Types.ObjectId, ref: "Message" }],
+  participants: [{ type: mongoose.Types.ObjectId, required: true, ref: "User" }],
+  created_date: { type: Date, default: Date.now},
+  message_date: { type: Date, default: Date.now},
+  
+});
```

Comment:
Can you run prettier on this to add some line breaks, it's a bit cramped right now

Reply:
Done!